packages/@react-spectrum/numberfield/docs/NumberField.mdx

```diff
@@ -0,0 +1,216 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/numberfield';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/numberfield/package.json';
+
+```jsx import
+import {NumberField} from '@react-spectrum/numberfield';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Forms
+keywords: [number field, input]
+---
+
+# NumberField
+
+<p>{docs.exports.NumberField.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['NumberField']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<NumberField label="Name" />
+```
```

Comment:
Prob not super important and not a blocker for this pull, but I noticed that the left border of the numberfield seems a bit wider than normal:
In docs:
![image](https://user-images.githubusercontent.com/9661127/106821683-79099080-6632-11eb-8475-0c5147182531.png)

In storybook:
![image](https://user-images.githubusercontent.com/9661127/106821717-8888d980-6632-11eb-87a1-5c87f1fe3ce7.png)

Reply:
Nvm, pulled in main and it is resolved. However, there seems to be a specificity issue (at least on my local) where the stepper css isn't properly overriding the textfield width styles:
broken:
![image](https://user-images.githubusercontent.com/9661127/106822989-064de480-6635-11eb-9517-519ef9e1cc9c.png)

if the styles are properly overridden:
![image](https://user-images.githubusercontent.com/9661127/106823062-1cf43b80-6635-11eb-992e-2c23541ea82c.png)


Prob just need to make `.spectrum-Stepper-field` more specific (like adding .spectrum-Stepper in front)